models/queries/contributors.js

```diff
@@ -1,9 +1,11 @@
 const contributorsQueries = {
   contribute:
-    'INSERT INTO contributions (amount, owner) VALUES ($1, $2) RETURNING *',
-  getContrubutions: 'SELECT * FROM contributions WHERE owner=$1',
-  getContrubutionById: 'SELECT * FROM contributions WHERE id=$1',
-  payContribution: 'UPDATE contributions SET status = 1, paiddate = $1 WHERE id = $2 AND status = 0 RETURNING *',
+    "INSERT INTO contributions (amount, owner) VALUES ($1, $2) RETURNING *",
+  getContrubutions: "SELECT * FROM contributions WHERE owner=$1",
+  getLoans: "SELECT * FROM loans ",
```

Comment:
I think the total amount requested has to be for loans which are pending only, what do you think?

Reply:
You are right. let me fix that.

Reply:
It has been fixed.